pysb/bng.py

```diff
@@ -49,14 +52,25 @@ class BngBaseInterface(object):
 
     @abc.abstractmethod
     def __init__(self, model=None, verbose=False, cleanup=False,
-                 output_prefix=None, output_dir=None):
+                 output_prefix=None, output_dir=None,
+                 model_additional_species=None,
+                 model_population_maps=None):
+        self._logger = get_logger(__name__,
+                                  model=model,
+                                  log_level=verbose)
+        if model:
```

Comment:
logging.py's `get_logger` creates a `PySBModelLoggerAdapter` if a model is supplied, or a `Logger` otherwise. For most purposes the API for an Adapter and a Logger is similar, but it doesn't have a `getEffectiveLevel()` method. `self._logger.logger` is the underlying Logger object when `self._logger` is actually an adapter, not a logger. I'll add a comment to clarify this.